src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -53,9 +54,23 @@ const ListEvent = ({ item: feed, theme, styles }: FeedEventProps) => {
         <View style={styles.transferInfo} alignItems="flex-start">
           <View style={styles.mainInfo}>
             <EventCounterParty style={styles.feedItem} feedItem={feed} />
-            <Text numberOfLines={1} style={styles.message}>
-              {feed.data.message}
-            </Text>
+            {feed.type === 'welcome' ? (
```

Comment:
This should be data from the event feed item (example a link for read more so we show the read more if there is a link. 
However, if the intent is just to show a mock this can make the trick.